### PR TITLE
Update jsonpatch.md

### PR DESCRIPTION
**Pull request by ishchenkoviacheslav**
just a typo i guess.

### CODE REVIEW

**Comment by pranavkm** [comment]
Isn't this the JSONPatch:

```
  {
    "op": "move",
    "from": "/orders/1",
    "path": "/orders/0"
  }
```

The current description looks correct. Why do you think this needs to be changed?

File: aspnetcore/web-api/jsonpatch.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ The following sample patch document:
 
 * Copies the value of `Orders[0].OrderName` to `CustomerName`.
 * Sets `Orders[0].OrderName` to null.
-* Moves `Orders[0]` to before `Orders[1]`.
```
Comment (ishchenkoviacheslav) [inline aspnetcore/web-api/jsonpatch.md]: Sorry it was my fault. Wrong line.

File: aspnetcore/web-api/jsonpatch.md [hunk 2]
```diff
@@ -218,7 +218,7 @@ This operation is functionally the same as a `move` operation without the final
 The following sample patch document:
 
 * Copies the value of `Orders[0].OrderName` to `CustomerName`.
-* Inserts a copy of `Orders[1]` before `Orders[0]`.
```
Comment (ishchenkoviacheslav) [inline aspnetcore/web-api/jsonpatch.md]: here, i guess you should change description for patch or change patch.

Comment (pranavkm) [reply aspnetcore/web-api/jsonpatch.md]: While this is correct, I think a better change would be to change the sample to do this:

```
{
    "op": "copy",
    "from": "/orders/1",
    "path": "/orders/0"
  }
```

It keeps it similar to the Move operation and would make this comment correct

Comment (ishchenkoviacheslav) [reply aspnetcore/web-api/jsonpatch.md]: totally agree.

Comment (ishchenkoviacheslav) [reply aspnetcore/web-api/jsonpatch.md]: have created one other PR.

**Comment by pranavkm** [comment]
Closing this in favor of the other PR.

**Pull request closed without merging**